### ISSUE TITLE
Update button labeling

### ISSUE THREAD

**Issue by joedolson** [ableplayer/ableplayer#659]
Right now, Able Player's buttons use an `aria-label`, internal screen reader text, and have a visible tooltip generated from the `aria-label`. 

This goes back to #81, because JAWS wasn't able to read buttons without innerText. (E.g., didn't support `aria-label`. This is no longer true, and this should be simplified.

The simplest change is to eliminate the `able-clipped` container inside buttons. It might be better to offer support for auto translation by retaining screen reader text; I'm just not sure it's worth it. The package is already translated into dozens of languages, so this would be an edge scenario. 

Right now, however, the `able-clipped` container does not serve any purpose; it's overridden by the `aria-label`.

**Comment by joedolson** [comment]
Closed by https://github.com/ableplayer/ableplayer/commit/1d14cbf567602c9e1f89e8241c276901e1beaaea
